docs/pages/versions/unversioned/react-native/safeareaview.md

```diff
@@ -11,6 +11,8 @@ The purpose of `SafeAreaView` is to render content within the safe area boundari
 
 To use, wrap your top level view with a `SafeAreaView` with a `flex: 1` style applied to it. You may also want to use a background color that matches your application's design.
```

Comment:
```suggestion
To use, wrap your top level view with a `SafeAreaView` with a `flex: 1` style applied to it, so it fills all of the available space. You may also want to use a background color that matches your application's design.
```

Let's expand already existing sentence, instead of adding a new note.